tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -183,7 +183,7 @@ set( test_src
   WaterThermalTanks.unit.cc
   WaterToAirHeatPump.unit.cc
   WaterToAirHeatPumpSimple.unit.cc
-  WaterToWaterHeatPumpEIR.unit.cc
+        PlantLoopHeatPumpEIR.unit.cc
```

Comment:
Indentation snuck in here, I know CLion likes to indent the CMakeLists files automatically...